src/core/ngx_connection.c

```diff
@@ -78,6 +78,7 @@ ngx_create_listening(ngx_conf_t *cf, struct sockaddr *sockaddr,
 
     ls->fd = (ngx_socket_t) -1;
     ls->type = SOCK_STREAM;
```

Comment:
no test to adapt?

Reply:
Tests are hosted [here](http://hg.nginx.org/nginx-tests) as a mercurial repo, but the documentation does not mention the addition of new tests. I would assume that we are not expected to add any to the suite.